### PR TITLE
Sql

### PR DESCRIPTION
**Pull request by auvipy**

### CODE REVIEW

File: .gitreview [hunk 1]
```diff
@@ -1,4 +1,4 @@
 [gerrit]
 host=gerrit.sqlalchemy.org
 project=sqlalchemy/sqlalchemy
-defaultbranch=master
+defaultbranch=rel_2_0
```
Comment (auvipy) [inline .gitreview]: is it correct?

**Review by auvipy — COMMENTED** [review]
will fix the merge conflicts

File: .gitreview (hunk 1 above)
Comment (zzzeek) [reply .gitreview]: if the PR is against master then that should stay at "master".

**Comment by zzzeek** [comment]
So I need PRs that have just one or two things changed each, against current master.  These PRs look like they are getting all confused in merging across branches or something.   I know it's a little tricky to get git to do the right thing.

**Comment by auvipy** [comment]
Ok, I am going to come with smaller chunks of changes on top of master. Will close this after coming with that. Allow me a lil bit of time.

**Comment by auvipy** [comment]
are you done with those?

**Comment by zzzeek** [comment]
the 2.0 branch is not really ready right now as a lot of what I was going to put in 2.0 I'm actually putting in 1.4.  I'm going to keep getting as much of the infrastructure into 1.4 as I can without major backwards incompatibility.  then i *think* what I will try first is to build out some of sqlalchemy.future in 1.4, that will be the basis for things to go into 2.0.    I'm still not really sure how I want to organize 1.4  / 2.0 branches because I'm concerned about changes being easy to port between the two.

**Comment by zzzeek** [comment]
in 1.4 / master, I do want to remove older things that are deprecated, anything marked deprecated before 1.1 or so we can just remove at this point.   but if you contribute they have to be small PRs so they can be managed more easily.

**Comment by auvipy** [comment]
OK, i will be sending once deprecation once, on 1.4

**Comment by auvipy** [comment]
Many small PR's are coming to address this! against master

**Pull request closed without merging**